app/controllers/api/teams.php

```diff
@@ -453,7 +453,7 @@
         if (empty($invitee)) { // Create new user if no user with same email found
             $limit = $project->getAttribute('auths', [])['limit'] ?? 0;
 
-            if ($limit !== 0 && $project->getId() !== 'console') { // check users limit, console invites are allways allowed.
+            if (!$isPrivilegedUser && $limit !== 0 && $project->getId() !== 'console') { // check users limit, console invites are allways allowed.
```

Comment:
This is great! Let's also add `!$isAppUser` too so that it's still possible to create a user with an API key.

```suggestion
            if (!$isPrivilegedUser && !$isAppUser && $limit !== 0 && $project->getId() !== 'console') { // check users limit, console invites are allways allowed.
```

Reply:
I accidently clicked on commmit

Reply:
No, We cannot add, If its AppUser, he should not be able to add more user if count increases

Reply:
```
if (!$isPrivilegedUser && $limit !== 0 && $project->getId() !== 'console') { // check users limit, console invites are allways allowed.
                $total = $dbForProject->count('users', [], APP_LIMIT_USERS);

                if ($total >= $limit) {
                    throw new Exception(Exception::USER_COUNT_EXCEEDED, 'Project registration is restricted. Contact your administrator for more information.');
                }
 }
```

The if condition checks if its privileged, it would not check the limit but if it is app user it should check it

Reply:
app user means API key. API key should not be restricted by this.